migrate.py

```diff
@@ -259,6 +259,12 @@
 	result			BOOL NOT NULL
 );
 
+CREATE TABLE IF NOT EXISTS linus_rants (
```

Comment:
Table names are normally in singular form (`linus_rant`, `rant` or just `linus`)

Reply:
Very nice that you're using a table to do this though!